.eslintrc.js

```diff
@@ -0,0 +1,66 @@
+module.exports = {
+    "env": {
+        "browser": true,
+        "es6": true,
+        "jquery": true,
+        "jasmine": true
```

Comment:
We don't have any JS tests right now. We will probably use jasmine if/when we do to be inline with bedrock but it should be removed for now.